content/post/wrong-data.md

```diff
@@ -0,0 +1,55 @@
+
+## It really is wrong
+If you are giving your screen the “what the heck” stare, and find out that you goofed in your code, give yourself a pat on the back -- seriously. It’s easy enough to write the code and ship it, but to sanity check it and have the business context to see numbers don’t pass the sniff test takes practice. Catching “wrong” because you know what the numbers should be is an accomplishment. 
+
+You can up your chances at catching wrong data here by finding test groups and present your findings to them. 
+The data is confusing (but not wrong)
+This explains most of the “wrong” data I’ve seen, and pops up a lot when the business logic is hidden behind some sort of reporting layer or business intelligence tool. If you have a sales report, does it count returns? Once you issue the refund you have not made anything from the purchase, and you might still have the item in your warehouse (depending on when it was refunded). These metrics require pristine documentation because they can mean so many things to so many people. To a marketer and a warehouse manager words like “order” and “revenue” can call to mind different business logic. If you are providing reporting, you need to find the language unifing those departments. 
+
+I have seen website reporting tools cause problems here, too. Most tools don’t show real-time data in reports. So pulling up traffic details for your site might show you data is lagging hours behind. If you thought a new part of your site was launching and your website dashboard shows near-zero traffic going to it, you are going to want answers quick. 
+
+Notice the problem here is in how the data is presented or taught to the data consumer. Just sending the data in an email, or building a report can do more harm than good.
+
+## Teach the customer; fix the product
+Finding the reason someone thinks a data point is wrong is key to applying the right solution, but It should not be used to offload responsibility. If data consumers look at a report and get the wrong idea of the business logic under the hood, that is as much of a problem as the math being wrong in a function. 
+
+Reporting tools are great for a business (see: [_Reporting is a Gateway Drug_](https://www.locallyoptimistic.com/post/reporting-is-a-gateway-drug/)), but with sharing information comes the reponsibility to help people make sense of it, and to have the right expectations. If you are digging into the data every day with code you’re all too familiar with the imperfections of the data stores you are touching. It’s helpful to keep clarifying what words mean, and what the data is tracking under the hood as you present it. For instance, if you talk about site sessions, share in writing what exactly a site session is. Sure everyone understands the concept of a session, but do they all have the same definition as the site tracking tool you’re using for your analysis? 
```

Comment:
I think it'd be better not to end with a question -- maybe re-work the paragraph so you lead with the question and end with the recommendation / answer.

Reply:
This also might be a good spot to mention the [data quality flywheel](https://www.locallyoptimistic.com/post/data-dies-in-darkness/) (and give me that sweet sweet backlink)

Reply:
I feel like we need a summary section at the end, maybe under a `## Conclusions` header that ties everything up nicely.

Reply:
done, not pushed yet.